react/the_react_ecosystem/styling_react_applications.md

```diff
@@ -58,4 +58,5 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- [Tailwind CSS documentation](https://tailwindcss.com/)
+- [Tailwind CSS documentation](https://tailwindcss.com/).
+- We recommend checking out this article that goes [very in-depth on some amazing things](https://www.joshwcomeau.com/css/styled-components/)https://www.joshwcomeau.com/css/styled-components/ you can do using styled components. 
```

Comment:
```suggestion
- Josh Comeau has a nice article on [getting the most out of styled-components](https://www.joshwcomeau.com/css/styled-components/). 
```
I think the article is a good read and also reasonable considering we have the Tailwind docs in the additional resources.

I do not think we should be telling learners that "we recommend reading this article" in this case:
- It's an additional resource. We should not be recommending that people read something which we state is not required. And I don't think the article is appropriate for an assignment.
- Given that TOP still for the most part recommends as much vanilla CSS as possible and these are only options of varying abstractions from vanilla CSS (e.g. CSS modules is almost the same but is very slightly different), we should not be using wording that suggests TOP recommending the use of any other particular option.

We ideally should just, especially for additional resources, state what the resource is and what it's for then let the learner decide if it's relevant enough to read.

Reply:
@nikitarevenco Just checking in in case you haven't seen this! Let me know if there are any issues.

Reply:
Oh thanks! I must've forgotten about this. I have added your suggestions but it seems that there aren't any linting issues here. What I am seeing instead is some files that are unrelated to this PR and the failing checks for them. 

![image](https://github.com/TheOdinProject/curriculum/assets/154856872/488e6e3b-e92e-4a89-98df-ff6fcc410858)

Reply:
Usually there would be errors here. For example you can see that the backtick is used insead of a single quote on line 59. But the linter doesn't seem to be flagging that?

![image](https://github.com/TheOdinProject/curriculum/assets/154856872/66e66ba4-fc84-4c7b-ac75-d421ca4a8808)

Reply:
I'm not entirely sure why the linter isn't picking that specific error up, given that your `patch-69` branch looks like it does contain the TOP003 rule config.

As for the check annotations stuff, that's just a super annoying beta feature of GitHub that we're not sure exactly why it's happening. But sometimes with older PRs, when an update is made, it runs the actions on unrelated files to try and be helpful....but it really isn't helpful here and is just noise.

You can ignore those.